windirstat/Item.cpp

```diff
@@ -1390,6 +1404,7 @@ std::wstring CItem::UpwardGetPathWithoutBackslash() const
         estSize += p->m_Name.length() + 1;
     }
 
+    VTRACE(L"pathParts: TID {:>5} Size {:>3} Capacity {:>3}", GetCurrentThreadId(), pathParts.size(), pathParts.capacity());
```

Comment:
I can see why you wanted to add this for testing, but let's not include it in the final code since it's a trivial check.